youtube_dl/extractor/spankbang.py

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+import re
+
+class SpankBangIE(InfoExtractor):
+    """Extractor for http://spankbang.com"""
+    
+    _VALID_URL = r"https?://(?:www\.)?spankbang\.com/(?P<id>\w+)/video/.*"
+
```

Comment:
No, sorry.
I think the  ``stream_key`` is dynamic. I'm not sure how to test for that or if it would be worth it.